src/libraries/System.Runtime/tests/System/Text/RuneTests.cs

```diff
@@ -50,8 +50,6 @@ public static void Casing_CultureAware(int original, int upper, int lower, strin
         [InlineData('\u0130', '\u0130', '\u0130')] // U+0130 LATIN CAPITAL LETTER I WITH DOT ABOVE
         [InlineData('\u0131', '\u0131', '\u0131')] // U+0131 LATIN SMALL LETTER DOTLESS I
         [InlineData('\u1E9E', '\u1E9E', '\u1E9E')] // U+1E9E LATIN CAPITAL LETTER SHARP S
-        [InlineData(0x10400, 0x10400, 0x10428)] // U+10400 DESERET CAPITAL LETTER LONG I
```

Comment:
Should we move this to `MemberData` and condition these two test cases to run when `!NlsGlobalization`

Reply:
@GrabYourPitchforks can advise about that as I am not sure why he added this test cases to Nls only.

Reply:
Probably laziness on my part. :)

Reply:
will add it then :-)